src/libs/deploy.js

```diff
@@ -178,11 +180,131 @@ export const deployAppleStore = async (signedApp, workspace = '/tmp/') => {
 /**
  * AirWatch Deployment
  *
- * @param {String} signedApp The name of the signed app
+ * @param {String} job The name of the signed app
+ * @param {String} platform Define if it's android or iOS app
+ * @param {String} awOrgID The airwatch organization group ID to distribute the app
+ * @param {String} awFileName The name of the app to appear in airwatch
  * @param {string} [workspace='/tmp/'] The workspace to use
  * @returns The status of the deployment
  */
-// eslint-disable-next-line no-unused-vars
-export const deployAirWatch = async (signedApp, workspace = '/tmp/') => {
-  // TODO
+// eslint-disable-next-line import/prefer-default-export
+export const deployAirWatch = async (signedApp, platform, awOrgID, awFileName, workspace = '/tmp/') => {
+  /*
+  Values for airwatch api v8_1:
+    DeviceType: android -> '5'; Apple -> '2'
+    ModelId: android -> 5; iPhone -> 1; iPad -> 2
+    ApplicationName: android -> apk; Apple -> ipa
+  */
+
+  let deviceType = '';
+  let applicationName = '';
+  let modelId = 0;
+
+  switch (platform) {
+    case 'ios':
+    {
+      applicationName = awFileName + '.ipa';
+      deviceType = '2';
+      modelId = 1;
+      break;
+    }
+    case 'android':
+    {
+      applicationName = awFileName + '.apk';
+      deviceType = '5';
+      modelId = 5;
+      break;
+    }
+    default:
+      throw new Error('Unsupported application type for airWatch deployment');
+  }
+
+  // The constant url for api:
+  const awHost = 'https://mobileconsole.gov.bc.ca';
```

Comment:
These also might be better placed at the top of the function, top of the module, or in with config / constants.